### PR TITLE
add file

### PR DESCRIPTION
**Pull request by Curis-lab**
take some change

### CODE REVIEW

File: file.tsx [hunk 1]
```diff
@@ -0,0 +1,4 @@
+console.log('hello');
+
+console.log('take some file');
+console.log('take some change');
```
Comment (BrowserLoser) [inline file.tsx]: Could you make more better?

**Review by BrowserLoser — APPROVED** [review]
Would you make more fixable?

File: file.tsx (hunk 1 above)
Comment (Curis-lab) [reply file.tsx]: Which Line?

Comment (BrowserLoser) [reply file.tsx]:
```js
+  console.log('take some change');
- console.log('take some action');
```

**Pull request merged**